NICE.Identity.Management/ClientApp/src/views/OrganisationsList/OrganisationsList.tsx

```diff
@@ -0,0 +1,451 @@
+import React, { Component } from "react";
+import { Link } from "react-router-dom";
+import Moment from "moment";
+import { Breadcrumbs, Breadcrumb } from "@nice-digital/nds-breadcrumbs";
+import { Button } from "@nice-digital/nds-button";
+import { Card } from "@nice-digital/nds-card";
+import { Grid, GridItem } from "@nice-digital/nds-grid";
+import { PageHeader } from "@nice-digital/nds-page-header";
+import {
+	queryStringToObject,
+	appendQueryParameter,
+	removeQueryParameter,
+	stripMultipleQueries,
+} from "../../utils/querystring";
+import { fetchData } from "../../helpers/fetchData";
+import { isDataError } from "../../helpers/isDataError";
+import { Endpoints } from "../../data/endpoints";
+import { HistoryType, OrganisationType } from "../../models/types";
+import { FilterSearch } from "../../components/FilterSearch/FilterSearch";
+import { ErrorMessage } from "../../components/ErrorMessage/ErrorMessage";
+import { Pagination } from "../../components/Pagination/Pagination";
+
+import styles from "./OrganisationsList.module.scss";
+
+type CardMetaData = {
+	label?: string;
+	value: React.ReactNode;
+};
+
+type sortOption = {
+	identifier: string;
+	label: string;
+};
+
+type OrganisationsListProps = {
+	location: {
+		search: string;
+	};
+	history: HistoryType;
+};
+
+type OrganisationsListState = {
+	path: string;
+	organisations: Array<OrganisationType>;
+	searchQuery?: string;
+	error?: Error;
+	isLoading: boolean;
+	pageNumber: number;
```

Comment:
Storing things in state here feels maybe a bit odd considering we're also updating the URL via the history API. I wonder if we moved `pageNumber`, `itemsPerPage` and `sortedBy` into querystrings, it would mean we could use `history.push` or whatever the react-router way of doing it is, or even better, use `<Link>` to update the URL. This would mean we abstract bits like pagination etc into separate components which could update the URL as a way of getting the parent page to update. RR demos also contain a querystring demo, see https://v5.reactrouter.com/web/example/query-parameters - I think that approach is probably a bit nicer and it means we wouldn't need our custom methods like `queryStringToObject` etc.